src/tests/index.tsx

```diff
@@ -13,30 +20,24 @@ import { LocaleContext } from 'context/locale/localeContext/LocaleContext';
 
 import { ExtraRenderOptions, WrapperProps } from './types';
 
-const RouterForTests = ({ children, routerConfig }: WrapperProps) => {
-  if (routerConfig?.withRouter) {
-    // need to prefix the route with / for memory router
-    const initialEntries = routerConfig.routerHistory.map((historyItem) =>
-      historyItem.startsWith('/') ? historyItem : `/${historyItem}`,
-    );
-
-    const path = routerConfig.path.startsWith('/') ? routerConfig.path : `/${routerConfig.path}`;
-
-    return (
-      <Router initialEntries={initialEntries}>
-        <Routes>
-          <Route path={path} element={children} />
-        </Routes>
-      </Router>
-    );
-  }
-
-  return <Router>{children}</Router>;
-};
-
 // @TODO: https://bitbucket.org/thesoftwarehouse/react-starter-boilerplate/pull-requests/5/rss-9-add-login-page/diff#comment-132626297
-const _Wrapper = ({ children, routerConfig = { withRouter: false } }: WrapperProps) => {
+const _Wrapper = ({ children, routerConfig }: WrapperProps) => {
   const [locale, setLocale] = useState<AppLocale>(defaultLocale);
+  const { routerPath = '/', currentPath = routerPath } = routerConfig || {};
```

Comment:
My personal preference is to put a default value in props destructuring (on line 24) instead like it is done here using `|| {}` because thanks to that if you wanted to use routerConfig somewhere else in this component it would also be an empty object instead of `undefined`